bundles/com.salesforce.bazel-java-sdk/src/main/java/com/salesforce/bazel/sdk/project/BazelProject.java

```diff
@@ -45,6 +45,10 @@
  * A BazelProject can contain one or more Bazel packages.
  */
 public class BazelProject {
+
```

Comment:
BazelProject cannot depend on IProject. That would break SDK abstraction.

Reply:
I thought that, too. Is there a document explaining which abstractions are needed and who is using it? BazelProject and classpath computations seems to be very IDE centric. Keeping it in the SDK is more costly then beneficial. Thus, I was thinking of moving it out of the SDK and back into the IDE world.

Reply:
Gunnar and I discussed this offline. In short:

We can rework bazel-eclipse repo into whatever it needs to be. Priority 1 is to make good tooling, Priority 2 was to have all of that code reusable outside of IDEs. But if 2 is getting in the way of 1, we can end that. There is a clone of the SDK in this repo: https://github.com/salesforce/bazel-java-sdk so the idea still lives on there.